libraries/core/build.gradle.kts

```diff
@@ -37,6 +37,9 @@ dependencies {
     api(libs.androidx.lifecycle.common)
     api(libs.compose.animation.core)
     api(libs.compose.runtime)
+    api(libs.compose.material)
+    api(libs.compose.ui.tooling)
+    api(libs.compose.ui.ui)
```

Comment:
This is a shortcut for not extracting transitions-related code for now. It def shouldn't live in core in the long run.